akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -776,6 +778,7 @@ Host: example.com
       }
 
       Await.result(binding.unbind(), 10.seconds)
```

Comment:
Any particular reason to keep this `Await` and change others to `.futureValue`?

Reply:
Not sure in this case, but generally: sadly `Await.result` is usually faster than `.futureValue` because `.futureValue` always polls. That might not always matter much, but is the reason why we still use `Await.result` in a lot of places even though `.futureValue` is a nicer API. https://github.com/scalatest/scalatest/pull/1794 avoids the polling for `.futureValue`.

Reply:
Good point, but let's keep it like this. Maybe we'll have to revive `awaitResult` with a more concise syntax.